lta/picker.py

```diff
@@ -14,14 +15,17 @@
 from .log_format import StructuredFormatter
 
 EXPECTED_CONFIG = [
+    "FILE_CATALOG_REST_TOKEN",
     "FILE_CATALOG_REST_URL",
     "HEARTBEAT_PATCH_RETRIES",
     "HEARTBEAT_PATCH_TIMEOUT_SECONDS",
     "HEARTBEAT_SLEEP_DURATION_SECONDS",
     "LTA_REST_TOKEN",
     "LTA_REST_URL",
     "PICKER_NAME",
-    "WORK_SLEEP_DURATION_SECONDS"
+    "WORK_RETRIES",
+    "WORK_SLEEP_DURATION_SECONDS",
+    "WORK_TIMEOUT_SECONDS"
```

Comment:
Can some of these have defaults?  Sane defaults means less cruft when we actually have to configure the service in production.

Reply:
Note that the way I did it so far was to have a second `EXPECTED_CONFIG2 = {}`, or some such.

Reply:
Sure. I'm not opposed to sane defaults.